CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# WIP 3.0.0 (TBD)
+
+- **Breaking** Migrated to androidx packages
```

Comment:
Why is this considered a breaking change? 🤔

Reply:
@ngsilverman I think users who didn't migrate to androidx can't use libraries migrated to androidx.

Reply:
Yes, I think it is all or nothing.

Reply:
I haven't found any information about that. What makes you think that's the case?

Reply:
Jetifier transforms old package names to new package names. But there is no Jetifier that transforms from new package names to old ones.

Reply:
Most probably. For example, setting epoxy adapter (which extends androidx packaged base `RecyclerView.Adapter`) to `RecyclerView` with old package name won't work because it expects an old package named `RecyclerView.Adapter`. Right?
Assuming user uses just `RecyclerView` not `EpoxyRecyclerView`.

Reply:
Ah, I see, makes sense for this use case. I'm also trying to figure out if it should be considered a breaking change for Paris (https://github.com/airbnb/paris) but it doesn't seem so since there's no input parameter type from the AndroidX library.

Reply:
You probably will need to test that out, if you didn't, by importing migrated Paris to old non-migrated sample app, as there might be weird clashes between AndroidX and old packages.